src/runtime-rs/crates/hypervisor/src/dragonball/inner_device.rs

```diff
@@ -209,9 +210,14 @@ impl DragonballInner {
         Ok(())
     }
 
-    fn add_net_device(&mut self, config: &NetworkConfig) -> Result<()> {
+    fn add_net_device(
+        &mut self,
+        hconfig: &HypervisorConfig,
```

Comment:
IMO, there's no need introduce a hconfig here, you can just get the hypervisor config in add_net_device before you do dragonball_network_device